packages/tools/devtools/devtools-core/src/CommonInterfaces.ts

```diff
@@ -29,23 +29,23 @@ export interface HasContainerKey {
 }
 
 /**
- * A unique ID for a Fluid object
+ * A unique key for a Fluid object
```

Comment:
Nit (while we're here)
```suggestion
 * A unique key for identifying a Fluid object associated with a Container registered with Devtools.
```